Example/AztecUITests/AztecUITests.swift

```diff
@@ -0,0 +1,263 @@
+        richContentTextView.enterAndSelectText(text: "text")
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.unorderedlistButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<ul><li>text</li></ul>"
+        XCTAssertEqual(expected, text)
+    }
+
+    // Enable this test after ordered lists are fully implemented
+    func testSimpleOrderedListText() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
+        richContentTextView.tap()
+        richContentTextView.enterAndSelectText(text: "text")
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.orderedlistButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<ol><li>text</li></ol>"
+        XCTAssertEqual(expected, text)
+    }
+
+    func testSimpleLinkedText() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
+        richContentTextView.tap()
+        richContentTextView.enterAndSelectText(text: "text")
+
+        // Copy link to be auto-filled in URL field
+        UIPasteboard.general.string = "https://wordpress.com/"
+        app.scrollViews.otherElements.buttons[elementStringIDs.linkButton].tap()
+        app.alerts.buttons.element(boundBy: 3).tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<a href=\"https://wordpress.com/\">text</a>"
+        XCTAssertEqual(expected, text)
+    }
+
+    func testHorizontalRuler() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
+        richContentTextView.tap()
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.horizontalrulerButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let tex: String = htmlContentTextView.value as! String
+        let expected = "<hr>"
+        XCTAssertEqual(expected, text)
+    }
+
+    func testMoreTag() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
+        richContentTextView.tap()
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.moreButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<!--more-->"
+        XCTAssertEqual(expected, text)
+    }
+
+    func testHeadingOneText() {
```

Comment:
Agreed. My plan for this PR was to implement the most basic tests for adding the correct style to text, but we should add the header tests you've outlined as well.

We should also have similar tests for blockquotes and lists (to test that the style ends when expected and is only applied to the expected text), as well as combining styles appropriately.

I've made a note to revisit these in a future PR.